.devcontainer/.zshrc

```diff
@@ -110,25 +110,31 @@ if [ "$(stat -c '%U' /usr/local/share/npm-global)" != "node" ]; then sudo chown
 
```

Comment:
This first file is an orthogonal change that only effects developers using a VS Code Dev Container (AFAIK, that's just me.)

The change puts me on the same version of 'fluid-build' that everyone else is using instead of building/using a private fluid-build.

The motivation is that it will hopefully help me to remember to deploy 'fluid-build' after making changes.